kubernetes/overlays/dev/prompt-proto-service/prompt-proto-service.yaml

```diff
@@ -15,7 +15,8 @@ spec:
       imagePullSecrets:
       - name: regcred
       containers:
-        - image: us-central1-docker.pkg.dev/prompt-proto/prompt/prompt-proto-service:latest
+        - image: us-central1-docker.pkg.dev/prompt-proto/prompt/prompt-proto-service:DM-36712
```

Comment:
Not sure it's a good idea to commit the ticket/branch number, but I guess we can fix that later.